test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -244,51 +252,78 @@ def assert_compatibilities_spectral_centroid(self, sample_rate, n_fft, hop_lengt
 
         self.assertEqual(out_torch.type(out_librosa.dtype), out_librosa, atol=1e-5, rtol=1e-5)
 
-    def test_basics1(self):
-        kwargs = {
-            'n_fft': 400,
-            'hop_length': 200,
-            'power': 2.0,
-            'n_mels': 128,
-            'n_mfcc': 40,
-            'sample_rate': 16000
+    def test_basic_compatibilities(self):
+        compat_tests = {
+            'assert_compatibilities_core': self.assert_compatibilities_core,
+            'assert_compatibilities_mel_spectogram': self.assert_compatibilities_mel_spectogram,
+            'assert_compatibilities_s2db': self.assert_compatibilities_s2db,
+            'assert_compatibilities_mfcc': self.assert_compatibilities_mfcc,
+            'assert_compatibilities_spectral_centroid': self.assert_compatibilities_spectral_centroid,
         }
-        self.assert_compatibilities(**kwargs)
-
-    def test_basics2(self):
-        kwargs = {
-            'n_fft': 600,
-            'hop_length': 100,
-            'power': 2.0,
-            'n_mels': 128,
-            'n_mfcc': 20,
-            'sample_rate': 16000
-        }
-        self.assert_compatibilities(**kwargs)
-
-    # NOTE: Test passes offline, but fails on TravisCI (and CircleCI), see #372.
-    @unittest.skipIf('CI' in os.environ, 'Test is known to fail on CI')
-    def test_basics3(self):
-        kwargs = {
-            'n_fft': 200,
-            'hop_length': 50,
-            'power': 2.0,
-            'n_mels': 128,
-            'n_mfcc': 50,
-            'sample_rate': 24000
-        }
-        self.assert_compatibilities(**kwargs)
-
-    def test_basics4(self):
-        kwargs = {
-            'n_fft': 400,
-            'hop_length': 200,
-            'power': 3.0,
-            'n_mels': 128,
-            'n_mfcc': 40,
-            'sample_rate': 16000
+        params_list = [
+            {
+                'n_fft': 400,
+                'hop_length': 200,
+                'power': 2.0,
+                'n_mels': 128,
+                'n_mfcc': 40,
+            },
+            {
+                'n_fft': 600,
+                'hop_length': 100,
+                'power': 2.0,
+                'n_mels': 128,
+                'n_mfcc': 20,
+            },
+            {
+                'n_fft': 400,
+                'hop_length': 200,
+                'power': 3.0,
+                'n_mels': 128,
+                'n_mfcc': 40,
+            },
+            {
+                'n_fft': 200,
+                'hop_length': 50,
+                'power': 2.0,
+                'n_mels': 128,
+                'n_mfcc': 50,
+                'skip_CI': True,  # NOTE: Test passes offline, but fails on TravisCI (and CircleCI), see #372.
+            },
```

Comment:
Please move the parameters outside of test definition by using `parameterized.expand`